### PR TITLE
[WIP] `Reskin` - text italicized on iPhone WTF?

### PR DESCRIPTION
**Pull request by mudrila**

### CODE REVIEW

**Comment by tomstuart123** [comment]
I can't really test this as I never was able to replicate the 'italics'. @azadikoff / @decentdao/design could one of you who was seeing it prior, test it out?

**Comment by adamgall** [comment]
I'm still seeing this on my iPhone. Also, attempting to debug and fix now too.

File: index.html [hunk 1]
```diff
@@ -136,7 +136,7 @@
       font-style: normal;
       font-weight: 400;
       src:
-        url('/fonts/TT_Firs_Neue_Variable.ttf') format('truetype'),
+        url('/fonts/TT-Firs-Neue-Variable.ttf') format('truetype'),
```
Comment (DarksightKellar) [inline index.html:139]: Lmaooo how did you even discover this?

Comment (adamgall) [reply index.html]: I Pay Attention To Details.

**Comment by xraystyle1980** [comment]
I’m still not seeing the background color in Firefox mobile. Italics went away though 👍
<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Firefox 125.3 on iOS 17.4.1
Viewport:  393 x 793 @3x
Language:  en-US
Cookies:   Disabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-1780.app.dev.fractalframework.xyz%2F&os=iOS&os_version=17.4&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-1780.app.dev.fractalframework.xyz/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6643814c04e74000089dcddd&commentID=6643965e8f3c18092df43830&resolution=resolved)

**Pull request merged**